src/MediaBundleForm.php

```diff
@@ -92,7 +126,11 @@ public function form(array $form, FormStateInterface $form_state) {
           ),
         ),
       );
-      $form['type_configuration'][$plugin] += \Drupal::service('plugin.manager.media_entity.type')->createInstance($plugin, $plugin_configuration)->settingsForm($this->entity);
```

Comment:
Another \Drupal::service('plugin.manager.media_entity.type') call above that could leverage injected dependency now.